### ISSUE TITLE
Niagara Button shortcut for opening a new Chrome tab not working correctly

### ISSUE THREAD

**Issue by uicrafts** [8bitPit/Niagara-Issues#1619]
**Describe the bug**
Using the (beta) Niagara button, I've assigned it a shortcut to open a new Chrome tab on tab or swipe up. It opens Chrome, but on the first opened tab, instead of a brand new one.

**To Reproduce**
1. Enable the Niagara button
2. Assign the "open new Chrome tab" shortcut on tap or swipe up
3. Chrome will open on an existing tab, usually the first tab on the list of open tabs
4. **What should happen: Chrome opens in a brand new tab, and the focus is put on the search bar automatically**

**Smartphone (please complete the following information):**
 - Device: Google Pixel 5 (stock Android, not rooted)
 - Android version: Android 11 (stable), May update

**Comment by 8bitPit** [comment]
Thank you very much for reporting, @uicrafts. Does the same bug also occur when adding the shortcut in another third-party launcher (e.g., Nova Launcher or Microsoft Launcher)?

**Comment by uicrafts** [comment]
@8bitPit -- I tried both Microsoft and Nova launchers, and I couldn't find a shortcut specifically to open in a new Chrome tab. However, using Niagara, over the last few days using the swipe up gesture on the button seems to launch a new Chrome tab instead of the first already-opened one. The only thing that changed is that I switched the shortcut to open an Edge tab, then back to Chrome, and it worked. Would be nice if the focus was put on the URL bar, but I'm not sure if that would be a feature request or a bug.
